05-classification.Rmd

```diff
@@ -963,12 +988,14 @@ for the number of neighbours and weight function (for now, these are just the va
  manually when we created `knn_spec` above). But now the fit object also includes information about
 the overall workflow, including the centering and scaling preprocessing steps.
 
-Let's visualize the predictions that this trained K-nearest neighbour model will make on new observations.
-Below you will see how to make the coloured prediction map plots from earlier in this chapter.
+Figure \@ref(fig:05-workflow-plot-show) visualizes the predictions that this 
```

Comment:
This should be a separate section I think. To clearly indicate it is not part of the workflow part, and to indicate this is "bonus" or "optional" material.

Reply:
the code should be optional, and i'll make that clear. but looking at the visualization should not be